WordPress/Classes/ViewRelated/Stats/Insights/Latest Post Summary/LatestPostSummaryCell.swift

```diff
@@ -51,6 +51,14 @@ class LatestPostSummaryCell: UITableViewCell, NibLoadable {
         applyStyles()
     }
 
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        rowsStackView.arrangedSubviews.forEach {
+            rowsStackView.removeArrangedSubview($0)
+            $0.removeFromSuperview()
+        }
+    }
+
```

Comment:
There is actually a helper method in `UITableViewCell+Stats` that does exactly this. You should be able to replace the `.forEach` with `removeRowsFromStackView(rowsStackView)`. i.e:
```
    override func prepareForReuse() {
        super.prepareForReuse()
        removeRowsFromStackView(rowsStackView)
    }
```

Reply:
thanks for this!